src/components/pre-playback-play-overlay/_pre-playback-play-overlay.scss

```diff
@@ -32,6 +32,14 @@
   }
 }
 
+.pre-playback {
+  .bottom-bar,
+  .top-bar {
+    opacity: 0 !important;
+    display: none;
```

Comment:
Why do we need both opacity=0 and display=none?

Reply:
good Q. this is how it was before. I guess `opacity` for the transition and `display` for cross browsers (see #33)

Reply:
@yairans so lets understand this and remove if not needed - having important seems a bit rough